.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: groovy
```

Comment:
What's actually building here. Only the tests?

Reply:
Adding groovy also adds support for Java (and apparently Kotlin 🤷 ), so everything builds